tests/networkpolicy_test.go

```diff
@@ -181,19 +181,19 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("[test_id:CNV-1515] should be failed to reach vmia from vmic", func() {
+		It("[test_id:1515] should be failed to reach vmia from vmic", func() {
 			By("Connect vmia from vmic")
 			ip := vmia.Status.Interfaces[0].IP
 			assertPingFail(ip, vmic)
 		})
 
-		It("[test_id:CNV-1516] should be failed to reach vmia from vmib", func() {
+		It("[test_id:1516] should be failed to reach vmia from vmib", func() {
```

Comment:
@dshchedr The test case description says opposite of test steps. Would you please take a look?

Reply:
Fixed.